src/main.rs

```diff
@@ -185,6 +188,16 @@ fn settings() -> io::Result<Settings> {
                 .number_of_values(1)
                 .help("Prevents deletion of images for which repository:tag matches <REGEX>"),
         )
+        .arg(
+            Arg::with_name(DELETION_CHUNK_SIZE_OPTION)
+                .value_name("DELETION CHUNK SIZE")
+                .short("d")
+                .long(DELETION_CHUNK_SIZE_OPTION)
+                .help(&format!(
+                    "Removes specified quantity of images before checking disk usage (default: {})",
```

Comment:
I think it would read a tiny bit better if we change `Removes specified quantity of images before checking disk usage` to `Removes specified quantity of images at a time` (and update the README accordingly).